README.md

```diff
@@ -17,4 +17,169 @@
+
+```js
+const animal = {
+  species: 'lion'
+};
+```
+
+Now we're talking. Take a moment to study the syntax a bit, and think about what we just did. We added a property to our object when we created it. If you remember, a property has both a *key* and a *value*. In this case, our object has a property with the key `species` and the string `'lion'` as its value.
+
+You might be way ahead of me already, but let's create a lion out of *thin air*, with some more relevant properties. Ready? Go!
+
+```js
+const simba = {
+  species: 'lion',
+  name: 'Simba',
+  weight: 190,
+  age: 8
+};
+```
+
+A value can be of any type. This means that we can store strings, numbers, booleans, arrays and functions in our object. We can even add other objects as values to our object! An image of [Leonardo Di Caprio looking at a gravity defying cityscape](https://www.wired.com/images_blogs/underwire/2010/07/inception_paris_660.jpg) springs to mind, but I digress. 
+
+```js
+const simba = {
+  species: 'lion',
+  name: 'Simba',
+  weight: 190,
+  age: 8,
+  diet: ['antelope', 'tortoise', 'zebra'],
+  personality: {
+    friendly: true,
+    aggressive: false
+  },
+  roar: function() {
+    console.log('Meow.');
+  }
+};
+```
+
+
+## Access granted
+Now that we have a data structure representing our favorite lion, let's poke and prod a little. The data's in there, but how do we get it out? How do we access the values of the properties?
+
+We can do that in two ways: the *dot notation*, or the *bracket notation*. Unless you have something funky going on with your key names (i.e. special characters and whatnot), or you want to access properties on an object in a dynamic way using variables, you will almost always use the dot notation.
+
+This is what using the dot notation looks like:
+
+```js
+console.log(simba.species); // prints 'lion'
+console.log(simba.diet); // prints ['antelope', 'tortoise', 'zebra']
+simba.roar(); // prints 'Meow.'
+```
+
+Again, you will mostly use this notation to access properties on an object, so it's a good thing it's so easy to write! Now, what if, for some reason, we wanted special characters in our property keys, like a space or a weird symbol? First, to define a key with a special character, we need to add quotes around the key. As you know, single or double quotes both work. Use whatever tickles your fancy, but usually people use single quotes for this.
+
+```js
+const simba = {
+  'animal type': 'lion'
+};
+```
+
+To print this property, we'd have to use the bracket notation instead of the dot notation. This is because if we were to type `console.log(simba.animal type);`, it would get interpreted in the wrong way. We need to nudge JS in the right direction:
```

Comment:
Could we have students try this? Give them a feel for how something like this fails.